jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerSession.kt

```diff
@@ -250,7 +250,7 @@ class CodeModernizerSession(
     /**
      * This will resume the job, i.e. it will resume the main job loop kicked of by [createModernizationJob]
      */
-    fun resumeJob(startTime: Instant) = state.putJobHistory(sessionContext, "Started", startTime)
+    fun resumeJob(startTime: Instant, jobId: JobId) = state.putJobHistory(sessionContext, "Started", jobId.id, startTime)
```

Comment:
The status in the putJobHistory function is in upperCase [here](https://github.com/aws/aws-toolkit-jetbrains/pull/4087/files#diff-8b0ea0f291f1c20406e922c82de118911f271b383adaa9520b9b3bf521abaf43R119). Should this be changed to ensure parity? Is it a good idea to make this an enum instead of a string?

Reply:
great find, will fix this